README.md

```diff
@@ -235,10 +235,10 @@ A few advantages of this solution:
 
 ## Develop
 
-1. Make sure you are using Node.js 14 or higher and npm 7 or higher
-1. Write code and tests.
-1. Make sure all tests pass `npm test`
-1. Make sure code is well formatted and secure `npm run lint`
+1. Make sure you are using Node.js 15 or higher and npm 7 or higher.
```

Comment:
does it mean we drop support for Node 14? isn't it a breaking change?

Reply:
On the one hand it is, but on the other it is not. You can always have node 14 but npm in 7 version - you have to install npm manually.

Reply:
So a breaking change yes 😄

Reply:
Would it make sense to release this in the 2.0 version of the parser then? We could say "we support >=16 but it might work in older versions" kinda message.